gen/input/src/main/scala/test/StarWarsSubquery.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+// format: off
+/*
+  rules = [GraphQLGen]
+  GraphQLGen.schemaDirs = ["gen/input/src/main/resources/graphql/schemas"]
+ */
+package test
+
+import clue.GraphQLSubquery
+import test.StarWars
+
+@clue.annotation.GraphQLSubquery
+trait StarWarsSubquery extends GraphQLSubquery[StarWars] {
+
+  override val rootType: String = "Character"
+
+  override val subquery: String = """
+        {
+          name
+        }
+      """
+}
+
+@clue.annotation.GraphQLStub
+object StarWarsSubquery
+// format: on
```

Comment:
Subqueries are written like this.

Reply:
Btw, it's a bit annoying that we need to define both the `trait` and the `object` but otherwise we don't have a way to refer to the subquery within the same compilation unit.